Sources/main.swift

```diff
@@ -0,0 +1,75 @@
+import HTTPServer
+import Router
+import JSON
+
+func getIdParameter(from request: Request) -> Int {
+    guard let idString = request.pathParameters["id"] else { return 0 }
+    return Int(idString) ?? 0
+}
+
+let router = Router { route in
+
+    var todos = [JSON?]()
```

Comment:
Instead of representing a Todo as just `JSON`, I think it's a much better idea to make a `Todo` struct which, in conjunction with [Mapper](https://github.com/zewo/mapper) and ContentNegotiationMiddleware, will parse and serialize automatically (while still being type-safe)